src/modules/SourceManager.rb

```diff
@@ -580,7 +581,7 @@ def AddUpdateSources(urls)
           ret = Builtins.add(ret, url) # #180820#c26
 
           # is there any patch available?
-          patches = Pkg.ResolvableProperties("", :patch, "")
+          patches = Y2Packager::Resolvable.find(kind: :patch)
```

Comment:
Just use `Resolvable.any?` here, we just detect if there is any patch available at all.

Reply:
But we are also logging how many patches are available.